### ISSUE TITLE
Counts not showing up for getValues?

### ISSUE THREAD

**Issue by k----n** [ssc-oscar/lookup#7]
The #ofCommitIds, #ofFiles, #ofCommits, #ofBlob-IDs, #ofProjects, #ofCommit-IDs, etc. don't show up when running `~/lookup/getValues`.

This is different than the README which suggests the counts exist.

What changed to cause this?
Is it because the counts are no longer in the maps?

**Comment by audrism** [comment]
Having counts makes running the filters in sequence more complicated, e.g to compose 
map yyy and zzz
```
echo XXX | getValue -f yyy | cut -d\; -f2 | lsort 1G -u | getValue -f zzz
``` 
so README needs to be updated to reflect current behaviour.

**Comment by audrism** [comment]
Fixed in 45df0a043e76a360cad188088439d40fdee11d78
